configure.d/1_blk_queue_exit_for_rhel.conf

```diff
@@ -12,7 +12,7 @@ LINUX_MAJOR_SINCE=4
 LINUX_MINOR_SINCE=18
 LINUX_SUB_SINCE=0
 RHEL_MAJOR_SINCE=348
-RHEL_MINOR_SINCE=2
```

Comment:
maybe prefix these variables names with "QUEUE_PUT_WA_" or something similar to indicate what this is about